webservices/common/models.py

```diff
@@ -595,3 +595,49 @@ class CommitteeTotalsIEOnly(BaseModel):
     coverage_end_date = db.Column(db.DateTime)
     total_independent_contributions = db.Column(db.Integer)
     total_independent_expenditures = db.Column(db.Integer)
+
+
+class Filings():
+    __tablename__ = 'vw_filing_history'
+
+    committee_id = db.Column(db.String)
+    sub_id = db.Column(db.BigInteger, primary_key=True)
+    coverage_start_date = db.Column(db.DateTime)
+    coverage_end_date = db.Column(db.DateTime)
+    receipt_date = db.Column(db.DateTime)
+    election_year = db.Column(db.Integer)
+    form_type = db.Column(db.String)
+    report_year = db.Column(db.Integer)
+    report_type = db.Column(db.String)
+    # document_type = db.Column(db.String)
```

Comment:
After the email, they are making the suggested changes, so when they do we want to uncomment the new models and delete the old model below. They made the change on their end but I don't see it on our side yet.